### ISSUE TITLE
[Bug][a11y] In "Notifications" the "Firefox News" options state is not read

### ISSUE THREAD

**Issue by Diana-Rus** [mozilla-mobile/fenix#14813]
## Prerequisites:
- Have Talk Back enabled
- Have a clean profile
## Steps to reproduce
1. Open app and on go to Menu->Settings->Notifications.
2. Tap on the "Firefox news" option.
### Expected behavior
"Firefox new-Make sound on-switch" is read by talk back.
### Actual behavior
"Firefox new-Make sound double tap to activate" is read by talk back.
### Device information

* Android device: Samsung Galaxy S9 (Android 8)
* Fenix version: Nightly 9/7

**Comment by Diana-Rus** [comment]
hi, yup sure

<details closed><summary>Screen</summary>

![Screenshot_20200907-151227_Settings](https://user-images.githubusercontent.com/49859863/92386695-f03b0700-f11c-11ea-8eb3-2d34da999e1b.jpg)
</details>
So this can't be really changed than, right?

**Comment by mcarare** [comment]
Yes, this is a list of notification channels , the screen is an OS screen, that we cannot change.

**Comment by Diana-Rus** [comment]
Thank you, i'll be closing the issue 😊
